### ISSUE TITLE
Server crashes (NullPointerException)

### ISSUE THREAD

**Issue by koooh** [Rammelkast/AntiCheatReloaded#12]
Hi!
Since I added this plugin yesterday, my server crashed three times with the same error message.
I disabled it today and had no new crash.

```
[14:36:01] [ERROR] Encountered an unexpected exception
net.minecraft.server.v1_10_R1.ReportedException: Ticking block entity
at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:810) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:400) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:668) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:567) [minecraft_server.jar:git-Spigot-570526c-c1b529e]
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
Caused by: java.lang.NullPointerException
at net.minecraft.server.v1_10_R1.EntityPlayer.a(EntityPlayer.java:954) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.EntityLiving.addEffect(EntityLiving.java:660) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.TileEntityBeacon.applyEffect(TileEntityBeacon.java:124) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.TileEntityBeacon.E(TileEntityBeacon.java:146) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.TileEntityBeacon.m(TileEntityBeacon.java:77) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.TileEntityBeacon.E_(TileEntityBeacon.java:69) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.World.tickEntities(World.java:1496) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.WorldServer.tickEntities(WorldServer.java:632) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:799) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
... 4 more
[14:36:01] [ERROR] Cause of unexpected exception was
java.lang.NullPointerException
at net.minecraft.server.v1_10_R1.EntityPlayer.a(EntityPlayer.java:954) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.EntityLiving.addEffect(EntityLiving.java:660) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.TileEntityBeacon.applyEffect(TileEntityBeacon.java:124) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.TileEntityBeacon.E(TileEntityBeacon.java:146) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.TileEntityBeacon.m(TileEntityBeacon.java:77) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.TileEntityBeacon.E_(TileEntityBeacon.java:69) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.World.tickEntities(World.java:1496) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.WorldServer.tickEntities(WorldServer.java:632) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:799) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:400) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:668) ~[minecraft_server.jar:git-Spigot-570526c-c1b529e]
at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:567) [minecraft_server.jar:git-Spigot-570526c-c1b529e]
at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
```

**Comment by Rammelkast** [comment]
That doesn't look good :O, can you put the entire log on pastebin?

**Comment by koooh** [comment]
Sure, here's the whole crash report: http://pastebin.com/eBs59Dhu
AntiCheatReloaded doesn't appear in the stacktrace but I confirm the crashes only happen when the plugin is loaded.

**Comment by Rammelkast** [comment]
These are some next level errors... 
[Help Wanted]

**Comment by Rammelkast** [comment]
I believe i fixed this issue, together with #15 in commit fade626.
If this happens again, please re-open this ticket.
